### PR TITLE
feat: generate tests with env files

### PR DESCRIPTION
**Pull request by vsukhin**
## Checklist (choose whats happened)

- [ ] breaking change! (describe)
- [x] tested locally
- [ ] tested on cluster
- [ ] added new dependencies
- [x] updated the docs
- [x] added a test

## Breaking changes

-

## Changes

-

## Fixes

-

### CODE REVIEW

**Review by exu — APPROVED** [review]

File: docs/generate-test-crds.md [hunk 1]
```diff
@@ -0,0 +1,61 @@
+# File naming convention for test CRD generation
```
Comment (jfermi) [inline docs/generate-test-crds.md:1]:
```suggestion
# File Naming Convention for Test CRD Generation
```

File: docs/generate-test-crds.md [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+
+We support following file naming convention for Postman files to support multple environment files:
```
Comment (jfermi) [inline docs/generate-test-crds.md:3]:
```suggestion
We support the following file naming convention for Postman files to support multiple environment files:
```

File: docs/generate-test-crds.md [hunk 3]
```diff
@@ -0,0 +1,61 @@
...
+
+## Test filename convention
```
Comment (jfermi) [inline docs/generate-test-crds.md:5]:
```suggestion
## Test Filename Convention
```

File: docs/generate-test-crds.md [hunk 4]
```diff
@@ -0,0 +1,61 @@
...
+
+<Test name>.postman_collection.json - where `Test name` should be reused for environments files.
```
Comment (jfermi) [inline docs/generate-test-crds.md:7]:
```suggestion
<Test name>.postman_collection.json - where `Test name` should be reused for the environment files.
```

File: docs/generate-test-crds.md [hunk 5]
```diff
@@ -0,0 +1,61 @@
...
+
+For example, mytest.postman_collection.json
+
+## Test environment filename convention
```
Comment (jfermi) [inline docs/generate-test-crds.md:11]:
```suggestion
## Test Environment Filename Convention
```

File: docs/generate-test-crds.md [hunk 6]
```diff
@@ -0,0 +1,61 @@
...
+
+<Test name>.<Test env>.postman_environment.json - where `Test name` is reused from test files and
```
Comment (jfermi) [inline docs/generate-test-crds.md:13]:
```suggestion
<Test name>.<Test env>.postman_environment.json - where `Test name` is reused from the test files and 
```

File: docs/generate-test-crds.md [hunk 7]
```diff
@@ -0,0 +1,61 @@
...
+## Test filename convention
+
+<Test name>.postman_collection.json - where `Test name` should be reused for environments files.
+
+For example, mytest.postman_collection.json
+
+## Test environment filename convention
+
+<Test name>.<Test env>.postman_environment.json - where `Test name` is reused from test files and 
+`Test env` is pointing to particular testing environment.
```
Comment (jfermi) [inline docs/generate-test-crds.md:14]:
```suggestion
`Test env` is pointing to a particular testing environment.
```

File: docs/generate-test-crds.md [hunk 8]
```diff
@@ -0,0 +1,61 @@
...
+
+For example, mytest.prod.postman_collection.json
```
Comment (jfermi) [inline docs/generate-test-crds.md:16]:
```suggestion
For example, mytest.prod.postman_collection.json.
```

File: docs/generate-test-crds.md [hunk 9]
```diff
@@ -0,0 +1,61 @@
+# File naming convention for test CRD generation
+
+We support following file naming convention for Postman files to support multple environment files:
+
+## Test filename convention
+
+<Test name>.postman_collection.json - where `Test name` should be reused for environments files.
+
+For example, mytest.postman_collection.json
```
Comment (jfermi) [inline docs/generate-test-crds.md:9]:
```suggestion
For example, mytest.postman_collection.json.
```

File: docs/generate-test-crds.md [hunk 10]
```diff
@@ -0,0 +1,61 @@
...
+
+## Test environment filename convention
+
+<Test name>.<Test env>.postman_environment.json - where `Test name` is reused from test files and 
+`Test env` is pointing to particular testing environment.
+
+For example, mytest.prod.postman_collection.json
+
+## Test secret environment filename convention
```
Comment (jfermi) [inline docs/generate-test-crds.md:18]:
```suggestion
## Test Secret Environment File Naming Convention
```

File: docs/generate-test-crds.md (hunk 5 above)
Comment (jfermi) [inline docs/generate-test-crds.md:11]:
```suggestion
## Test Environment File Naming Convention
```

File: docs/generate-test-crds.md [hunk 11]
```diff
@@ -0,0 +1,61 @@
...
+
+<Test name>.<Test env>.postman_secret_environment.json - where `Test name` is reused from test files and 
+`Test env` is pointing to particular testing environment.
+
+For example, mytest.prod.postman_secret_environment.json
```
Comment (jfermi) [inline docs/generate-test-crds.md:23]:
```suggestion
For example, mytest.prod.postman_secret_environment.json.
```

**Review by jfermi — COMMENTED** [review]
Editing suggestions.

**Pull request merged**